Lib/test/test_email/test_email.py

```diff
@@ -3319,15 +3319,90 @@ def test_getaddresses(self):
            [('Al Person', 'aperson@dom.ain'),
             ('Bud Person', 'bperson@dom.ain')])
 
+    def test_getaddresses_parsing_errors(self):
+        """Test for parsing errors from CVE-2023-27043"""
+        eq = self.assertEqual
+        eq(utils.getaddresses(['alice@example.org(<bob@example.com>']),
+           [('' ,'')])
```

Comment:
Similarly the lines below
```suggestion
           [('', '')])
```